keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/model/ActiveServerGroup.kt

```diff
@@ -74,7 +75,8 @@ data class ServerGroup(
   override val moniker: Moniker,
   override val buildInfo: BuildInfo? = null,
   override val disabled: Boolean,
-  override val instanceCounts: InstanceCounts
+  override val instanceCounts: InstanceCounts,
+  override val createdTime: Long
```

Comment:
FWIW Instant does not "just work", and since this is just in the clouddriver response I'm inclined to leave it as a long for ease of use. It doesn't feel worth it to transform it to another data type since that wouldn't give us anything.